### DISCUSSION TITLE
How to forward new "forum" messages?

### DISCUSSION THREAD

**Discussion by nisaev** [aahnik/tgcf#501] [Q&A]
Telegram now have "forum"(folders) in groups. How to forward it? And how to forward it to same structure groups?

**Comment by Eklip5e** [comment]
I suggest you to change the discussion title to "How to forward to/from new Supergroup Topics", as it has now a name given by Telegram

also,, quoted because I'd like to know the same thing

**Reply by noobietele** [reply]
Did you find any solution for topics support ??

**Reply by Eklip5e** [reply]
unfortunately not.
